### ISSUE TITLE
Why separate Styleguidist command?

### ISSUE THREAD

**Issue by nloding** [FelixKuehl/cra-monorepo#1]
I'm trying to make Styleguidist work with CRA2 in a monorepo for a current project. With CRA2 dropping lerna/monorepo support in the final release, I'm back on the hunt to try to make it work, and I found this repo. Looks like a pretty solid start so far.

One question - why a separate command to start styleguidist? Why not just run `styleguidist server` within the library you want to start it in? What was your thinking there?

**Comment by FelixKuehl** [comment]
Hey @nloding , 
First of all, thanks for commenting!
I had a few reasons to use a custom wrapper for the Styleguide server:
1. I wanted to share the exact same setup when using multiple libraries in the monorepo. When working with multiple libraries, applications and developers, I want to be able to adjust configurations (for example switching between `babel` and `nwb` for transpiling), without having to touch every single package. Building my own custom scripts, accumulates all configurations into the `library-utils` package. So every library uses the same version of babel, styleguidist, etc and their config.
2. When having a lot of packages that consume each other, it feels inconvenient having a terminal running the watcher and another one running the styleguide for every single package. So I wanted to join things into a single terminal application.
3. Having your own terminal process gives a lot of flexibility when you want to do custom jobs and / or run another process together with it. For example using `storybook` and `Styleguidist` together or running tests in parallel. 
4. I like clean output ;) 

However this is really just a starting point and to be improved.

**Comment by nloding** [comment]
I guess I don't see a difference in having scripts try to determine the best action vs. creating `"felix:rocks": "styleguidist server"` in one package and `"felix:rocks": "storybook-cli-cmd"` (whatever the storybook command is, if there is one?) then doing `lerna exec felix:rocks --parallel` ...

**Comment by FelixKuehl** [comment]
@nloding Yes agreed. But if you have multiple packages, sharing the configuration of your styleguide server and babel config, it might be convenient to have this in one way. You also do not need to have Styleguidist, Storybook, Babel, etc. installed in every individual package, when aggregating this into a single package. Makes version and config consistency a bit easier in my opinion. But also having a custom script might not be worth the effort for small use cases. My main Idea was, that `nathan:rules` should do the exact same thing in every package it is utilised. This makes it easy for new developers to understand your structure, which is an important aspect in my use case. And also (hopefully) increases maintainability.
